sbt-metals/src/main/scala/MetalsPlugin.scala

```diff
@@ -1,91 +1,91 @@
+package scala.meta.sbt
+
 import sbt._
 import sbt.Keys._
 import java.io._
-import sbt.Def
 
 object MetalsPlugin extends AutoPlugin {
   override def trigger = allRequirements
   override def requires = sbt.plugins.JvmPlugin
   object autoImport {
 
     val metalsCompilerConfig =
-      taskKey[String](
+      taskKey[Seq[(String, String)]](
```

Comment:
How about `Map[String, String]` to better convey the structure?

Reply:
I did that first, but sbt shows `Seq` nicer if you call `show metalsCompilerConfig`. On the other hand it's still not very readable, since values are huge. So I'll change it to Map again.